### PR TITLE
Re-add tests from #960

### PR DESCRIPTION
**Pull request by ceeram**

### CODE REVIEW

**Comment by ceeram** [comment]
Added back the tests from #960 to ensure this behavior is unchanged after the method name revert in #970

File: tests/AuthorizationServerTest.php [hunk 1]
```diff
@@ -107,9 +107,58 @@ public function testGetResponseType()
...
 
+        $responseTypeA = $method->invoke($server);
+        $responseTypeB = $method->invoke($server);
+        $this->assertInstanceOf(BearerTokenResponse::class, $responseTypeA);
+        $this->assertInstanceOf(BearerTokenResponse::class, $responseTypeB);
+        $this->assertNotSame($responseTypeA, $responseTypeB);
         $this->assertInstanceOf(BearerTokenResponse::class, $method->invoke($server));
     }
 
+    public function testGetResponseTypeStateless()
```
Comment (Sephster) [inline tests/AuthorizationServerTest.php]: This is a replication of the test below. I just changed the name

File: tests/AuthorizationServerTest.php [hunk 2]
```diff
@@ -107,9 +107,58 @@ public function testGetResponseType()
         $method = $abstractGrantReflection->getMethod('getResponseType');
         $method->setAccessible(true);
 
+        $responseTypeA = $method->invoke($server);
```
Comment (Sephster) [inline tests/AuthorizationServerTest.php]: I removed this section of the tests as it is replicated in the test below as far as I can tell.

**Comment by ceeram** [comment]
In this pr some asserts were removed, https://github.com/thephpleague/oauth2-server/pull/970/files#diff-257ce39e10ed4b9dcbbf1a46a2b4dab6L110

indeed not all of it, but some.

**Pull request closed without merging**